test/fixtures/minitest_spec_describe_with_classname.rb

```diff
@@ -0,0 +1,10 @@
+# typed: true
```

Comment:
Let's remove the `typed` sigils for fixtures, we don't have them in other places.